pkgs/development/python-modules/fastparquet/default.nix

```diff
@@ -12,15 +12,13 @@ buildPythonPackage rec {
     sha256 = "17i091kky34m2xivk29fqsyxxxa7v4352n79w01n7ni93za6wana";
   };
 
-  postPatch = ''
-    # FIXME: package zstandard
-    # removing the test dependency for now
-    substituteInPlace setup.py --replace "'zstandard'," ""
-  '';
-
   nativeBuildInputs = [ pytestrunner ];
   propagatedBuildInputs = [ numba numpy pandas thrift ];
-  checkInputs = [ pytest python-snappy lz4 zstd ];
+  checkInputs = [ pytestCheckHook python-snappy lz4 zstandard zstd ];
```

Comment:
Wouldn't be needed if the tests don't run. I suggest to keep it for now as upstream seems active and there is a chance that it get fixed in the near future.